lib/TPP/DefaultTppPasses.cpp

```diff
@@ -97,19 +98,35 @@ struct DefaultTppPasses
       // Bufferize: tensor->memref.
       pm.addPass(createBufferize());
 
-      // Lower all Tile operations.
-      pm.addNestedPass<func::FuncOp>(createLinalgLowering());
+      if (linalgToVector) {
+        pm.addNestedPass<func::FuncOp>(createVectorizationPass());
+        pm.addNestedPass<func::FuncOp>(createCanonicalizerPass());
+      } else {
+        // Lower all Tile operations.
+        pm.addNestedPass<func::FuncOp>(createLinalgLowering());
+      }
       pm.addPass(createCleanup());
     }
 
     // Convert forAll to parallel loops should run after bufferization
     // as scf.parallel does not handle tensor.
     pm.addPass(createConvertForAllToParallelOp());
-
-    // Low leve parallelization passes.
     LowLevelParallelizationOptions LowLevelParallelization{parallelTaskGrid};
-    pm.addPass(createLowLevelParallelization(LowLevelParallelization));
 
+    if (linalgToVector) {
```

Comment:
Please add a TODO here to disentangle this mess. We need to move away from heavy branching in the default pipeline or it will be hard to compose with other pipelines/transform schedules.